cuentociacua.txt

```diff
@@ -0,0 +1 @@
+Esta es la historia de dos niñas mimadas, prepotentes y fastidiosas que solo les gustaba hablar sobre lo que las otras niñas jugaban. Tanto lo hicieron que perdieron la alegriade vivir.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.